lib/components/Header.dart

```diff
@@ -19,6 +19,12 @@ class _IgniteHeaderState extends State<IgniteHeader> {
   Widget build(BuildContext context) {
     double width = MediaQuery.of(context).size.width;
     double adjustment = (width > VIEW_SIZE) ? (width - VIEW_SIZE) / 2 : 0;
+
+    if (width > 1000) {
```

Comment:
Hmm! Even more adjusted? Make me wonder what this page looks like on iPad. Would move this magic number to constants.dart, and maybe use a known break point number:

http://devfacts.com/media-queries-breakpoints-2020/

Looks like 992, 1024 (for iPads) or 1200